app/index.js

```diff
@@ -64,7 +66,20 @@ app.use(function errorHandler(err, req, res, next) {
   }
   next();
 });
-
+i18next.use(Backend).use(i18nextMiddleware.LanguageDetector).init({
+  backend: {
+    loadPath:'./locales/{{lng}}/razee-resources.json',
+    addPath:'./locales/{{lng}}/{{ns}}.missing.json'
```

Comment:
This generates a translation.missing.json file. Which we then never use.
Test it by putting `context.req.t('asdf')` somewhere in the code, running it, and seeing en/translation.missing.json. And then running `npm run i18n`, and still seeing the missing.json file still there.

Reply:
yes,missing.json file is created when there is a missing string.

Reply:
Yea, but we dont use that file. It just gets in the way, since `npm run i18n` populates the other json files properly and lets missing.json sit around